policy/cnspec_policy.proto

```diff
@@ -525,6 +525,9 @@ message ReportingJob {
   map<string,bool> datapoints = 7;
   map<string,cnquery.explorer.Impact> child_jobs = 9;
   Type type = 10;
+  // We will only store a ReportingJob by codeID
+  // But we add all the MRNs to it, so we can easily retrieve the MRN for a check
```

Comment:
Love that you are adding comments! let's make sure they are more targeted to the field, eg:
"MRNs of all checks / queries that this reporting job represents"

Reply:
Thanks for the feedback, changed this.